.azure-pipelines/ci-build.yml

```
@@ -12,235 +12,185 @@ pr:
-        displayName: 'ESRP CodeSigning Nuget Packages'
-        inputs:
-          ConnectedServiceName: 'microsoftgraph ESRP CodeSign DLL and NuGet (AKV)'
-          FolderPath: '$(Build.ArtifactStagingDirectory)'
-          Pattern: '*.nupkg'
-          signConfigType: inlineSignParams
-          inlineOperation: |
-            [
-                {
-                    "keyCode": "CP-401405",
-                    "operationSetCode": "NuGetSign",
-                    "parameters": [ ],
-                    "toolName": "sign",
-                    "toolVersion": "1.0"
-                },
-                {
-                    "keyCode": "CP-401405",
-                    "operationSetCode": "NuGetVerify",
-                    "parameters": [ ],
-                    "toolName": "sign",
-                    "toolVersion": "1.0"
-                }
-            ]
-          SessionTimeout: 20
-
-      - task: PublishPipelineArtifact@1
-        displayName: 'Upload Artifact: Nugets'
-        inputs:
-          artifactName: Nugets
-          targetPath: $(Build.ArtifactStagingDirectory)
-
-- stage: deploy
-  condition: and(contains(variables['build.sourceBranch'], 'refs/heads/master'), succeeded())
-  dependsOn: build
-  jobs:
-    - deployment: deploy
-      environment: nuget-org
-      strategy:
-        runOnce:
-          deploy:
-            pool:
-              vmImage: ubuntu-latest
-            steps:
-            - task: DownloadPipelineArtifact@2
-              displayName: Download nupkg from artifacts
-              inputs:
-                artifact: Nugets
-                source: current
-            - task: NuGetCommand@2
-              displayName: 'NuGet push'
-              inputs:
-                command: push
-                packagesToPush: '$(Pipeline.Workspace)/Nugets/Microsoft.OpenApi.OData.*.nupkg'
-                nuGetFeedType: external
-                publishFeedCredentials: 'OpenAPI Nuget Connection'
+                ]
+              SessionTimeout: 20
+
+    - stage: deploy
+      condition: and(contains(variables['build.sourceBranch'], 'refs/heads/master'), succeeded())
+      dependsOn: build
+      jobs:
+        - deployment: deploy
+          environment: nuget-org
+          strategy:
+            runOnce:
+              deploy:
+                pool:
+                  vmImage: ubuntu-latest
+                steps:
+                - task: DownloadPipelineArtifact@2
+                  displayName: Download nupkg from artifacts
+                  inputs:
+                    artifact: Nugets
+                    source: current
+                - task: 1ES.PublishNuget@1
+                  displayName: 'NuGet push'
+                  inputs:
+                    useDotNetTask: true
+                    packagesToPush: '$(Pipeline.Workspace)/Nugets/Microsoft.OpenApi.OData.*.nupkg'
+                    nuGetFeedType: external
+                    publishFeedCredentials: 'OpenAPI Nuget Connection'
```

```suggestion
                    publishFeedCredentials: 'OpenAPI Nuget Connection'
                    packageParentPath: '$(Pipeline.Workspace)'
```
See https://github.com/microsoft/OpenAPI.NET/pull/1676/files